### DISCUSSION TITLE
need custom fullScreenBackground not only system blur effect.

### DISCUSSION THREAD

**Discussion by lengyue524** [LucasMucGH/BottomSheet#129] [Ideas]
I want fullScreenBackground more customizable.
I want to set the fullScreenBackgroun color #000000 with opacity 0.75
Currently I can't set it.

**Comment by lucaszischka** [comment]
Hello,
I would like to get more input on this one, as we are currently not using colours with opacity but `Material` instead. This would therefore be a rather big change.
Greetings, Lucas :)

**Reply by lengyue524** [reply]
fullScreenBackground could be type of View which we can set any View as fullScreenBackground without limit in VisualEffect.

**Reply by lucaszischka** [reply]
Im not sure what you are trying to say there exactly, but you're free to open a pull request. However I will currently not work on this.
